lib/ProductOpener/Food.pm

```diff
@@ -4547,16 +4547,21 @@ sub compute_carbon_footprint_infocard($) {
 	
 	# Limit to France, as the carbon values from ADEME are intended for France
 	
```

Comment:
Could you add a comment explaining why are we removing those?

Reply:
This function computes those values, and at the start of the function we remove previous values if they exist. New values are computed below.

Reply:
@stephanegigandet I mean in the source code :-)